war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2408,25 +2407,24 @@ function createSearchBox(searchURL) {
         }
 
         copyFontStyle(box,sizer);
-        copyFontStyle(box,minW);
     });
 
     // update positions and sizes of the components relevant to search
     function updatePos() {
-        function max(a,b) { if(a>b) return a; else return b; }
-
         sizer.innerHTML = box.value.escapeHTML();
```

Comment:
I'd suggest we only run the contents of the `updatePos` function only when the window width changes. We should avoid running this function on vertical resizes.

Reply:
Disregard the above comment, that would only be the case if it runs on the window resize event, not on keyup